components/airtable/actions/create-multiple-records/create-multiple-records.js

```diff
@@ -0,0 +1,44 @@
+const airtable=require('../../airtable.app.js')
+const axios = require('axios')
+const chunk = require('lodash.chunk')
+const Airtable = require('airtable')
+
+module.exports = {
+  key: "airtable-create-multiple-records",
+  name: "Create Multiple Records",
+  description: "Create one or more records in a table by passing an array of objects containing field names and values as key/value pairs.",
+  version: "0.0.22",
+  type: "action",
+  props: {
+    airtable,
+    baseId: {type: "$.airtable.baseId", appProp: 'airtable'},
+    tableId: { type: '$.airtable.tableId', baseIdProp: 'baseId' },
+    records: { propDefinition: [airtable, "records"] },
+  },
+  methods: {
+    async addRecords(records) {
+      const base = new Airtable({apiKey: this.airtable.$auth.api_key}).base(this.baseId);
+      return await base(this.tableId).create(records)
+    },
```

Comment:
I would put this in the `.app.js` module, and keep any logic specific to the interaction against the Airtable API within that module, or at least add a method there that does this:

**airtable.app.js**
```js
const 
module.exports = {
  methods: {
    _apiKey() {
      return this.$auth.api_key;
    },
    getNewClient(baseId, tableId) {
      const clientOpts = {
        apiKey: this._apiKey(),
      },
      const airtableClient = new Airtable(clientOpts);
      return airtableClient.base(baseId)(tableId);
    },
  },
}; 
```